src/button-angular/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Angular CLI usually manages the `enableIvy` flag, no need to explicitly specify it. Although we don't recommend publishing Ivy instructions to npm right now, explicitly disabling it may have unexpected effect in the future.

Reply:
I think there was also an issue here with symbolic links.

Reply:
Removed.